internal/hcsoci/resources_lcow.go

```diff
@@ -22,7 +24,7 @@ func allocateLinuxResources(coi *createOptionsInternal, resources *Resources) er
 	if coi.Spec.Root == nil {
 		coi.Spec.Root = &specs.Root{}
 	}
-	if coi.Spec.Root.Path == "" {
+	if coi.Spec.Windows != nil && len(coi.Spec.Windows.LayerFolders) > 0 {
```

Comment:
We dont want this for docker's case where they do the mounting.

Reply:
Fair enough I didn't realize we forked it. What case is this ever going to not be true? A spec without these is invalid for LCOW

Reply:
Did you look at the else branch?

Reply:
I did: https://github.com/opencontainers/runtime-spec/blob/master/config.md


> `root` (object, OPTIONAL) specifies the container's root filesystem. On Windows, for Windows Server Containers, this field is REQUIRED. For Hyper-V Containers, this field MUST NOT be set.

Reply:
Because it works, and is useful for testing.